src/client/pages/OfferNew/Introduction/Sidebar/StartDate.tsx

```diff
@@ -313,10 +262,22 @@ const DateForm: React.FC<{
         size={size}
       >
         <Value>
-          <DateLabel hasStartDate={Boolean(dateValue)}>
-            {getDateLabel()}
-          </DateLabel>
-          {hasStartDate && <DownArrow />}
+          {isLoadingPickedStartDate && (
+            <LoadingDotsWrapper>
+              <LoadingDots color={colorsV3.gray500} />
+            </LoadingDotsWrapper>
+          )}
+          {!isLoadingPickedStartDate && (
+            <>
+              {!hasStartDate && hasCurrentInsurer(quote) && (
+                <StartDateLabelSwitcher
+                  dataCollectionId={quote.dataCollectionId}
+                />
+              )}
+              {hasStartDate && getDateLabel()}
+              <DownArrow />
+            </>
+          )}
```

Comment:
Sorry for being a pain over this 🙈  I think multiple conditions are easier to follow, like you had before, over nested if cases 😅

Reply:
Really? But this is the same principle as with a ternary operator - with the difference that the condition is clearly written out instead of just being "the thing that is not that other thing that we talked about som lines ago" (which it is with `:`/`else`).

Reply:
Maybe it's just med but yes, if choosing between those two options 🙃  But I'm fine with this 👍